bin/win/linkerd.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
shellcheck is parsing this file as a script, and we should add it to the list of ignored files in [static_check.yml](https://github.com/drholmie/linkerd2/blob/master/.github/workflows/static_checks.yml#L82).

`! -name *.nuspec \`

Reply:
Thanks for pointing this out! Will add it or `! -name win \` after resolving the previous comments.

Reply:
Sounds good! Looking forward to getting this committed and merged.